features/aanschrijfwijze.feature

```diff
@@ -474,22 +497,38 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
       Gegeven het systeem heeft een persoon met de volgende gegevens
```

Comment:
Zelfde opmerking hier als bij aanhef. Kunnen we dit niet in de RvIG implementatie oplossen, doordat de GbaPartner alleen de laatst ontbonden partner bevat in plaats van alle ontbonden partners? Of zijn er andere features die alle ontbonden partners nodig hebben? Dan krijgt de proxy altijd maximaal 1 ontbinden partner.